services-js/commissions-app/src/client/common/TextInput.stories.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import TextInput from './TextInput';
+
+storiesOf('TextInput', module).add('default', () => <TextInput />);
```

Comment:
Update this to send props to `<TextInput />`

You should also add a story to show off the “required” version.

See docs for Storybook: https://storybook.js.org/basics/writing-stories/

(Really this means chaining another call to `add` off of the one that’s already there.)

Reply:
TextInput.stories.tsx is already created so when you ask to add a story to show off required. Storybook is already showing the required <span>

Reply:
<span>

Reply:
span